### ISSUE TITLE
Fine-tuning EfficientLoFTR on a custom dataset

### ISSUE THREAD

**Issue by ChiaraLena** [zju3dv/EfficientLoFTR#78]
Hello,
Thank you for sharing your work!

I would like to fine-tune the model on a different dataset to adapt it to my specific application domain (the dataset structure is way different than MonoDepth, even though I have frames, relative depth maps, intrinsics and poses). However, I’m not entirely sure about the correct training procedure and configuration steps needed for this.

Could you please provide some guidance or best practices on the following points?
- Which parts of the code or config files should be modified to load a new dataset?
- Are there any particular preprocessing requirements (e.g., image normalization/size, depth or pose formats)?
- Any advice on hyperparameter adjustments when working with a smaller dataset?

Any suggestions would be greatly appreciated.

Thank you again!
Chiara
